rcnn/core/module.py

```diff
@@ -178,14 +178,11 @@ def forward(self, data_batch, is_train=None):
                 shape_changed = True
 
         if shape_changed:
-            module = Module(self._symbol, self._data_names, self._label_names,
-                            logger=self.logger, context=self._context,
-                            work_load_list=self._work_load_list,
-                            fixed_param_names=self._fixed_param_names)
+            self._curr_module._symbol = self._symbol
+            self._curr_module.binded=False
             module.bind(data_batch.provide_data, data_batch.provide_label, self._curr_module.for_training,
```

Comment:
"module" here seems to be an undefined variable. Did you mean "self._curr_module.bind"?

Reply:
Yes, It must bind before forward.
https://github.com/precedenceguo/mx-rcnn/blob/202fa9cf188a14d64e9a4e076a5b76b83374364a/rcnn/core/module.py#L160
It's bind here
https://github.com/precedenceguo/mx-rcnn/blob/202fa9cf188a14d64e9a4e076a5b76b83374364a/rcnn/core/module.py#L142